lib/compiler/swc/type-checker-host.ts

```diff
@@ -63,7 +63,7 @@ export class TypeCheckerHost {
     const reportWatchStatusCallback = (diagnostic: ts.Diagnostic) => {
       if (diagnostic.messageText !== TSC_NO_ERRORS_MESSAGE) {
         if ((diagnostic.messageText as string)?.includes('Found')) {
-          console.log(TSC_LOG_ERROR_PREFIX, chalk.red(diagnostic.messageText));
+          console.log(TSC_LOG_ERROR_PREFIX, red(diagnostic.messageText.toString()));
```

Comment:
Ansis method accepts the `string` or `number` argument, but `diagnostic.messageText` is `string | ts.DiagnosticMessageChain`, therefore we need it to convert into string